features/glance.feature

```diff
@@ -0,0 +1,39 @@
+@glance
+Feature: Glance
+  As an administrator
+  I want to make sure that Glance component is working correctly
+
+  @create_kvm_image
+  Scenario: Creation of KVM image
+    Given KVM source image exists
+    And glance image does not exist
+    When I create new glance image based on jeos1
+    Then the image is listed as active
+    And this image has non-empty ID
+    And its ID can be used to show the image info
+    When I delete the image
```

Comment:
Cleanup (i.e. test teardown) is different from actual testing.  Currently you are including deletion as part of the actual test.  Whilst I think it is right to properly test deletion, it should not be mixed in with testing of creation, because then if the test fails, you have to spend extra effort to determine whether it failed due to creation or deletion - it's basically confusing things by mixing multiple things into a single scenario.

In contrast, if you only want to delete as part of teardown, then it should be done outside the body of the test, e.g. via [an `After` hook for the scenario](https://github.com/cucumber/cucumber/wiki/Hooks#global-hooks).